src/lib/kdump/clients/kdump.rb

```diff
@@ -794,8 +788,7 @@ def cmdKdumpDumpLevel(options)
     def cmdKdumpDumpFormat(options)
       options = deep_copy(options)
       if Ops.get(options, "dump_format")
-        if Ops.get(options, "dump_format") == "ELF" ||
-            Ops.get(options, "dump_format") == "compressed"
+        if Set["none", "ELF", "compressed", "lzo", "snappy", "zstd", "raw"] === Ops.get(options, "dump_format")
```

Comment:
and here. Nice that you replace that `||`, but it ruby idiom to use here `.include?`